src/Data/Repositories/ChannelRepository.cs

```diff
@@ -145,7 +145,7 @@ public async Task<(bool, string?)> SafeRemove(Channel type, bool forceClose = fa
 
             if (!closeRequestAddResult.Item1)
             {
-                _logger.LogError("Error while saving close request for channel with id: {RequestId}", type.Id);
+                _logger.LogError("Error while saving close request for channel with id: {RequestId}: " + closeRequestAddResult.Item2, type.Id);
```

Comment:
Could it be something like this?
```suggestion
                _logger.LogError("Error while saving close request for channel with id: {RequestId}: {error}", closeRequestAddResult.Item2, type.Id);
```